.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,25 @@
+<!--
+In order for a pull request to be considered for inclusion in a release branch
+(release/x.y) after it has been cut, the pull request must have the following
+information filled out in its description.
+
+To draft a pull request using this template, append '&template=release.md' to
+a pull request preview URL (the "Open a pull request" page), or use the
+following URL template:
+
+https://github.com/apple/swift/compare/main...<username>:swift:<my-branch>?expand=1&template=release.md
+-->
+
+* Explanation: A description of the issue being fixed or enhancement being made.
+  This can be brief, but it should be clear.
+* Scope: An assessment of the impact/importance of the change. For example, is
+  the change a source-breaking language change, etc.
+* Issues: References to issues the change resolves, if any.
+* Original PR: Link to the main branch version of this pull request.
+* Risk: What is the (specific) risk to the release for taking this change?
+* Testing: What specific testing has been done or needs to be done to further
+  validate any impact of this change?
+* Reviewer: One or more code owners for the impacted components should review
```

Comment:
IIRC, I thought this was to mention the reviewer(s) who approved main branch PR as well. Should we mention in here? Or is that an useful information worth mentioning?

Reply:
Yeah, this is sketchy, so I just left it as is for now. Not every main branch PR is merged with (approving) reviews from code owners, and not every release branch PR is an atomic, full cherry-pick. The only reasonable thing this can mean that comes to mind is "A code owner or entitled delegate that has either already approved the main branch PR *if the release branch PR is identical*, or otherwise agreed to review both PRs". I do not see a point in having this entry unless the reviewer must be known at the time of opening the release branch PR.

@shahmishal What is this actual meaning of "Reviewer"?

Reply:
In practice this often means who reviewed the change in the main branch. As the approval on release branches in practice is often just the branch manager and the original reviewer doesn't necessarily review again. Tho preferably they would approve as well

Reply:
That much is clear 🙂 What I am wondering is if the authors of this form had other cases in mind. Like, who should it be if you merged the original PR in your own right or if there is no original PR? Logically, that would be someone who has agreed to review the possible back-port *beforehand*, but, if so, was this interpretation anticipated? Supposing it was, I think a regular review request is more sensible in this case because you don’t have to bother getting in touch with someone and arranging a review just to submit the PR. If that makes sense, then why have this entry at all if review info is derivable from the original PR in the other, predominant, case.